drivers/r2dbc-driver/src/main/kotlin/app/cash/sqldelight/driver/r2dbc/R2dbcDriver.kt

```diff
@@ -13,15 +13,47 @@ import kotlinx.coroutines.reactive.asFlow
 import kotlinx.coroutines.reactive.awaitFirstOrNull
 import kotlinx.coroutines.reactive.awaitSingle
 
-class R2dbcDriver(private val connection: Connection) : SqlDriver {
+class R2dbcDriver(private val connection: Connection, private val replaceParameter: Boolean) : SqlDriver {
```

Comment:
Good idea to disable auto replacement. If you use R2dbcDriver without sqldelight generated queries, it will be easier to use `executeQuery(null, "SELECT ABS($1)", {}, listOf(0)) { it.bindDouble(42) }` with some (useless) lists than counting the chars manually.

Reply:
The main reason for this change is the different behavior for different underlying databases. MySQL async tests will fail if `replaceParameter == true`. PostgreSQL on the other hand requires the parameters to be replaced. I would prefer it if this would be set depending on the dialect, but that would move the choice back into the code generation.